pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[tool.mypy]
+exclude = ['core/build/.*']
+files = 'core'
```

Comment:
This is actually less restrictive than now.  We currently restrict to `core/dbt` in `.pre-commit-config.yaml`.  If we alter this to only look there, you no longer need the exclude.

Would it be possible to only define the `files` key here and remove it from `.pre-commit-config.yaml` so we don't have it defined in multiple places for mypy?

Reply:
> Would it be possible to only define the `files` key here and remove it from `.pre-commit-config.yaml` so we don't have it defined in multiple places for mypy?

Yeah, annoyingly mypy doesn't have a notion of "force exclude", where it excludes something even if explicitly passed it on the command line. And pre-commit passes file names. So it's still necessary to have the duplication unfortunately

Reply:
> This is actually less restrictive than now. We currently restrict to `core/dbt` in `.pre-commit-config.yaml`. If we alter this to only look there, you no longer need the exclude.

Nice, done